src/confluent_kafka/admin/__init__.py

```diff
@@ -46,6 +46,11 @@
                      ScramCredentialInfo,
                      ScramMechanism,
                      UserScramCredentialsDescription)
+from ._topic import (TopicDescription)  # noqa: F401
```

Comment:
Add `TopicDescription` and `DescribeClusterResult` to index.rst in the `Admin API` section